Jenkinsfile

```diff
@@ -32,7 +32,7 @@ pipeline {
 					
 					mvn -U clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
 					-Pbuild-individual-bundles -Ptest-on-javase-17 -Pbree-libs -Papi-check \
-					-Djava.io.tmpdir=$WORKSPACE/tmp -Dcompare-version-with-baselines.skip=true -Dproject.build.sourceEncoding=UTF-8 \
+					-Djava.io.tmpdir=$WORKSPACE/tmp -Dproject.build.sourceEncoding=UTF-8 \
```

Comment:
Is this intentional? That would cause merge issue with https://github.com/eclipse-jdt/eclipse.jdt.core/pull/135/files

Reply:
this was intentional but I thought I add this change to https://github.com/eclipse-jdt/eclipse.jdt.core/pull/135/files

Reply:
added to wrong pr. Let me correct these